oneflow/user/ops/sparse_cross_entropy_op.cpp

```diff
@@ -22,12 +22,24 @@ namespace {
 
 Maybe<void> CheckPredictionLabelDesc(const user_op::TensorDesc* prediction_desc,
                                      const user_op::TensorDesc* label_desc) {
-  CHECK_EQ_OR_RETURN(prediction_desc->is_dynamic(), label_desc->is_dynamic());
-  CHECK_GE_OR_RETURN(prediction_desc->shape().NumAxes(), 2);
+  CHECK_EQ_OR_RETURN(prediction_desc->is_dynamic(), label_desc->is_dynamic())
+      << Error::RuntimeError()
+      << "Expected both prediction and label to be dynamic or neither, but found "
+      << prediction_desc->is_dynamic() << " and " << label_desc->is_dynamic();
+  CHECK_GE_OR_RETURN(prediction_desc->shape().NumAxes(), 2)
+      << Error::RuntimeError()
+      << "The dimension of prediction must greater than or equal to 2, but got "
+      << prediction_desc->shape().NumAxes();
   const int64_t num_out_axes = prediction_desc->shape().NumAxes() - 1;
-  CHECK_EQ_OR_RETURN(label_desc->shape().NumAxes(), num_out_axes);
+  CHECK_EQ_OR_RETURN(label_desc->shape().NumAxes(), num_out_axes)
+      << Error::RuntimeError()
+      << "Expected the dimension of label is one smaller than that of prediction, but got "
```

Comment:
The dimension of label is supposed to be less than that of prediction by 1, but found